pkg/app/piped/notifier/slack.go

```diff
@@ -150,10 +150,11 @@ func (s *slack) buildSlackMessage(event model.NotificationEvent, webURL string)
 			{"Started At", makeSlackDate(d.CreatedAt), true},
 		}
 	}
-	generatePipedEventData := func(id, version string) {
+	generatePipedEventData := func(id, name, version string) {
 		link = webURL + "/settings/piped"
 		fields = []slackField{
 			{"Id", id, true},
+			{"Name", name, true},
 			{"Version", version, true},
```

Comment:
I think `Id` should be the last one because it is the longest and less meaningful.

```suggestion
			{"Name", name, true},
			{"Version", version, true},
			{"Id", id, true},
```